src/test/java/org/fundacionjala/pivotal/testng/createWorkspaceTest.java

```diff
@@ -0,0 +1,48 @@
+package org.fundacionjala.pivotal.testng;
+
+import org.fundacionjala.core.ui.WebDriverManager;
+import org.fundacionjala.core.util.Environment;
+import org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard;
+import org.fundacionjala.pivotal.pageobjects.login.SignInPage;
+import org.fundacionjala.pivotal.pageobjects.workspaces.Workspaces;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class createWorkspaceTest {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00101)